### DISCUSSION TITLE
Want to revive my OVOS !

### DISCUSSION THREAD

**Discussion by 0xsharkboy** [OpenVoiceOS/ovos-buildroot#279] [General]
Hey,

I was interested by this project a long time ago but didn't really had the needed hardware. My touchscreen wasn't working well or partially on build root. I have an unused RPI 4 waiting for a new project 

So I wanted to ask you what would be your recommendation in terms of touchscreen, mic, speakers and potentially a case ?

Thank you!

**Comment by goldyfruit** [comment]
You current hardware should work with Raspberry Pi OS Bookworm Lite 64-bit and [ovos-installer](https://github.com/OpenVoiceOS/ovos-installer).
